packages/@internationalized/number/test/NumberParser.test.js

```diff
@@ -10,8 +10,13 @@
  * governing permissions and limitations under the License.
  */
 
+import fc from 'fast-check';
+import messages from '../../../@react-aria/numberfield/intl/*';
 import {NumberParser} from '../src/NumberParser';
 
+// for some reason hu-HU isn't supported in jsdom/node
+let locales = Object.keys(messages).map(locale => locale.replace('.json', '')).filter(locale => locale !== 'hu-HU');
```

Comment:
Nice, I was looking for a way to get them all! Clearly better than my hard-coded way